api-objects/UsersObject.ts

```diff
@@ -6,9 +6,10 @@ export enum AuthorityEnum {
 export const authorityTypes: Array<string> = ['Audiologist', 'Admin'];
 
 export class UsersObject {
+	public authorityid: number;
 	public username: string;
-	public name: string;
-	public email: string;
+	public authorityname: string;
```

Comment:
This change will break other services that use this object (such as my PR, which will be a merge conflict with this)